### PR TITLE
Add and update projects (backend)

### PR DESCRIPTION
**Pull request by ishaaa-ai**

### CODE REVIEW

File: server/src/routes.js [hunk 1]
```diff
@@ -8,4 +8,8 @@ module.exports = (app) => {
   app.post('/profile', ProfileController.register),
   app.post('/updateprofile', ProfileController.update_profile)
 
+  app.get('/home',ProfileController.projects);
+  app.post('/home',ProfileController.addProject);
```
Comment (VeerpalBrar) [inline server/src/routes.js]: why use '/home' and not '/project' ?

File: server/src/controllers/ProfilesController.js [hunk 2]
```diff
@@ -54,10 +54,38 @@ module.exports = {
     }
   },
 
-  /* This method updates the profile information of a user
-     with the new data provided by the user. The profile
-     information is updated with a given user id, by looking 
-     up the id in the database.  */
+  async addProject(req,res){
```
Comment (VeerpalBrar) [inline server/src/controllers/ProfilesController.js]: For clarity purposes, `ProfilesController.js` should contain all function related to creating and updating profiles. Let's create a `ProjectController.js`  and move the `addProject`, and `updateProject` functions into the file.  

This is pretty easy to do as you can copy `ProfileController.js` and rename the file and removes the profile related functions. In `routes.js` you will have to import in `ProjectController`  similar to below: https://github.com/itsninaricci29/CONNECTeD/blob/0e3e237e0fb28411f943a7e5bdab5036a8024665/server/src/routes.js#L1

**Comment by VeerpalBrar** [comment]
2 minor comments! Good work on setting up the backend!

File: server/src/routes.js (hunk 1 above)
Comment (ishaaa-ai) [reply server/src/routes.js]: I'll change it to post-projects.

File: server/src/controllers/ProfilesController.js (hunk 2 above)
Comment (ishaaa-ai) [reply server/src/controllers/ProfilesController.js]: Okay I'll make the file and copy these methods there. Thanks for the input :)

**Pull request closed without merging**